### PR TITLE
feat(ingest): Add support for user to delete the ingested items

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: scripts/superdesk-ingest/module.js [hunk 1]
```diff
@@ -1402,6 +1403,20 @@
...
+                controller: ['data', 'remove', function(data, remove) {
+                    remove.remove(data.item);
+                }],
+                filters: [
+                    {action: 'list', type: 'ingest'}
+                ],
+                additionalCondition:['remove', 'item', function(remove, item) {
+                    return remove.canRemove(item);
+                }],
+                privileges: {fetch: 1}
```
Comment (akintolga) [inline scripts/superdesk-ingest/module.js]: The server side privilege is ingest, here it is fetch.

Comment (ioanpocol) [inline scripts/superdesk-ingest/module.js]: I will change on backend

**Comment by mugurrus** [comment]
:+1:

**Comment by Aca-jov** [comment]
:+1:

**Comment by pavlovicnemanja** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**